README.md

```diff
@@ -20,15 +20,15 @@ Requirements
 --------------------------
 
 - Django 2.2
-- Python 3.5-3.7
+- Python 3.6-3.7
 
 
 Installation
 --------------------------
 
 1.  Install the app
     ```sh
-          $ pip install major-event-log
+          $ pip install django-major-event-log
```

Comment:
Do we want to maintain this app on pypi? Or should we change this to a clone and an install from the local copy?